ansible/playbooks/install_microk8s.yml

```diff
@@ -389,6 +389,14 @@
     tags:
       - helm_upgrade
 
+  - name: micork8s deploy | helm | configure chart values yaml
+    ansible.builtin.template:
+      src: ../../chart/values.yaml.j2
+      dest: "{{ browsertrix_cloud_home }}/chart/values.yaml"
+      mode: u+rw
+    tags:
+      - helm_upgrade
+
   - name: microk8s deploy | helm | deploy btrix (Debian)
     ansible.builtin.command: helm upgrade --install -f ./chart/values.yaml -f {{ microk8s_user_home }}/values.yaml btrix ./chart/
```

Comment:
Should this `{{ microk8s_user_home }}/values.yaml` and `{{ browsertrix_cloud_home }}/chart/values.yaml` be the same value? Which one is better one to use?

Reply:
Oh, I see, the above is having the template override the default, while this command supports multiple values files, which also support overriding (the values in 2nd one override values in 1st).
I think we need to decide on most flexible option here.